send-surveys/src/main/java/com/objectcomputing/pulsesurvey/model/Survey.java

```diff
@@ -0,0 +1,32 @@
+package com.objectcomputing.pulsesurvey.model;
+
+public class Survey {
```

Comment:
So...this doesn't really describe a "Survey".  I expect that we will eventually need a "Survey" entity.  I would call this "SendSurveysCommand".

Reply:
done